source/common.c

```diff
@@ -74,7 +74,7 @@ void *aws_mem_acquire(struct aws_allocator *allocator, size_t size) {
 }
 
 void *aws_mem_calloc(struct aws_allocator *allocator, size_t num, size_t size) {
-    AWS_PRECONDITION(allocator);
+    AWS_PRECONDITION(allocator, "Input pointer [allocator] mustn't be NULL.");
```

Comment:
You can remove the explanations here, but for clarity you could do `allocator != NULL` like you do in the post condition below.